Placeeo/Controller/MainViewController.swift

```diff
@@ -23,15 +24,25 @@ class MainViewController: UIViewController, MKMapViewDelegate{
     override func viewDidLoad() {
         super.viewDidLoad()
         view.addSubview(mapView)
-        
-        NSLayoutConstraint.activate([
-            mapView.leadingAnchor.constraint(equalTo: view.leadingAnchor),
-            mapView.trailingAnchor.constraint(equalTo: view.leadingAnchor),
-            mapView.topAnchor.constraint(equalTo: view.topAnchor),
-            mapView.bottomAnchor.constraint(equalTo: view.bottomAnchor)
-        ])
-        
+        title = "Home"
         mapView.show(places: places)
+        LocationManager.shared.getUserLocation{ [weak self] location in
+            DispatchQueue.main.async {
+                guard let strongSelf = self else {
+                    return
+                }
+                strongSelf.mapView.addPin(with: location)
+                LocationManager.shared.resolveLocationName(with: location, completion: { [weak self] locationName in self?.title = locationName})
```

Comment:
please follow the common code style described in https://github.com/raywenderlich/swift-style-guide